DEVELOPMENT.md

```diff
@@ -30,61 +43,60 @@ flowchart LR
 ```
 
 During test run, the test suites:
-1. create ephemeral pipelines and queues for a given [Buildkite Agent Cluster](https://buildkite.com/docs/clusters/overview).
-2. Run executor, which will monitor jobs from the target queue in target Buildkite Cluster,
-   starts new Jobs in a Kubernetes cluster.
-3. Test suite will clean up those ephemeral objects in the end.
+1. Create ephemeral pipelines and queues for a given [Buildkite Agent Cluster](https://buildkite.com/docs/clusters/overview).
+2. Run executor, which will monitor jobs from the target queue in target Buildkite Cluster and start new Jobs in a Kubernetes cluster.
+3. Clean up those ephemeral objects in the end.
```

Comment:
(Comment about the existing document) Not sure how this feels stylistically, but worth calling out which ephemeral objects are cleaned up? (The Kubernetes jobs and pods are also ephemeral objects, but not ones mentioned in step 1.)

```suggestion
5. Cleans up those ephemeral objects (pipelines and queues).
```